gp-easy-passthrough/gpep-edit-entry.php

```diff
@@ -44,70 +65,39 @@ public function update_entry_id( $entry_id, $form ) {
 		return $entry_id;
 	}
 
-	public function bypass_limit_submissions( $rule_groups, $form_id ) {
-
-		// Bypass GPLS if we're updating an entry.
-		if ( $this->get_update_entry_id( $form_id ) ) {
-			$rule_groups = array();
-		}
-
-		return $rule_groups;
+	public function get_passed_through_entries_input_name( $form_id ) {
+		return "gpepee_passed_through_entries_{$form_id}";
 	}
 
-	public function get_update_entry_id( $target_form_id ) {
+	public function get_passed_through_entry_ids( $form_id ) {
+		$entry_ids = explode( ',', rgpost( $this->get_passed_through_entries_input_name( $form_id ) ) );
```

Comment:
I really dig how simple the code is but I do fear that this could allow overwriting any entry in the database (or at least for the same form).

Reply:
@claygriffiths Thoughts on my new approach? I've added a hash of the passed-through entry IDs which is posted through with the entry IDs themselves. When we fetch them we validate that the hash matches before returning the posted entry IDs.